third_party/blink/renderer/modules/cobalt/h5vcc_system/h_5_vcc_system.idl

```diff
@@ -12,15 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+callback Callback = void();
```

Comment:
Could we call this something less generic, as we will likely have to define a callback type in other places? I'd either want the type to be defined in a more general place if it has a general name or have a more specific name if it's in a specific place.